SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -168,4 +168,6 @@ namespace SpatialConstants
 
 	const Worker_ComponentId MIN_EXTERNAL_SCHEMA_ID = 1000;
 	const Worker_ComponentId MAX_EXTERNAL_SCHEMA_ID = 2000;
+
+	const FString SPATIALOS_METRICS_DYNAMIC_FPS = TEXT("Dynamic.FPS");
```

Comment:
Any reason this needs to be an FString? Maybe just make it an std::string?

Reply:
Chatted with @improbable-valentyn with this. We currently do this for `ServerWorkerType` and `ClientWorkerType` when we don't need to and do the conversion when calling the WorkerSDK. 

I'm going to leave this as FString now to not break current convention but going to add a minor ticket to re-evaluate our use of strings here. Don't want to scope creep this PR with changing string usages.